hbase-asyncfs/src/test/java/org/apache/hadoop/hbase/io/asyncfs/TestFanOutOneBlockAsyncDFSOutput.java

```diff
@@ -72,14 +72,11 @@
     HBaseClassTestRule.forClass(TestFanOutOneBlockAsyncDFSOutput.class);
 
   private static final Logger LOG = LoggerFactory.getLogger(TestFanOutOneBlockAsyncDFSOutput.class);
-
   private static DistributedFileSystem FS;
-
   private static EventLoopGroup EVENT_LOOP_GROUP;
-
   private static Class<? extends Channel> CHANNEL_CLASS;
-
   private static int READ_TIMEOUT_MS = 2000;
+  private static final Random RNG = new Random();
```

Comment:
The one caveat is where the test does setSeed() TLR doesn't work because TLR doesn't support setSeed(). I don't know if this one of those cases or not. Some tests depend on setting the seed, HFile tests come to mind, and sometimes they don't. I can be more aggressive in switching to TLR where we don't really need to call setSeed. It's an option. YDYT @Apache9 ?

Reply:
setSeed could make the Random always give the same sequence, which could make the test more determined. So maybe for now let's just follow these rules:
1. By default, we will replace Random with ThreadLocalRandom.
2. If there is a setSeed call, then
  a. If the seed is just System.currentTimeMillis, then just replace it with ThreadLocalRandom.
  b. Otherwise, keep the old behavior, call setSeed on the Random object.

WDYT? Thanks.

Reply:
Yes @Apache9 that's what I've done :-)